Library/Homebrew/requirements.rb

```diff
@@ -114,10 +114,15 @@ def message
   end
 end
 
+class CVSRequirement < Requirement
+  fatal true
+  default_formula "cvs"
```

Comment:
Ignore my previous comment; it's right for a CVS formula to be used here 👍

Reply:
How so? This will force the formula even `MacOS::Xcode.provides_cvs?` is true.

Reply:
I think that's fine given how old those versions of macOS are at this point. I don't mind either way, though; if you'd rather this was handled by the Requirement: 👍 from me.

Reply:
I suppose someday our CVS formula might be something other than identical to Apple's.